tests/chaos/download-concurrent-large-files.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+###################################################################
+#Script Name	: download-large-files.sh
+#Description	: This is a script file that runs a curl command to download a file
+#Author       : ARG
+#Email        : 
+###################################################################
+
+. ../data/config/run.config
+
+echo '#####################################'
+echo `basename "$0"`
+echo '#####################################'
+echo ''
+
+for i in {0..3}
+do
+  nohup ./download-large-files.sh > logs/$i-download-large-files.log &
```

Comment:
I think this `download-large-file` approach will take way too long (also bandwidth if we're thinking of CI/CD on the cloud). A better approach would be to use:
```
sudo fallocate -l 2G bigfile
```
[source](https://unix.stackexchange.com/a/269221/235577)

Reply:
Ok. I'm ok using this but just an FYI that this assumes we have root access of the box.

Reply:
@jimmylee